package.json

```diff
@@ -6,67 +6,71 @@
   "scripts": {
     "prepare": "husky install",
     "cm": "cz",
-    "extension:build": "yarn workspace chrome-extension build",
-    "extension:zip": "yarn workspace chrome-extension zip",
-    "extension:watch": "yarn workspace chrome-extension watch",
-    "extension:e2e": "yarn workspace chrome-extension-e2e run e2e",
-    "extension:release": "yarn workspace chrome-extension release",
-    "extension:publish": "yarn workspace chrome-extension publish-to-store",
-    "website:build": "yarn workspace website build",
-    "website:start": "yarn workspace website start",
-    "website:release": "yarn workspace website release",
-    "build": "yarn website:build && yarn extension:build",
+    "extension:build": "pnpm --filter chrome-extension build",
+    "extension:zip": "pnpm --filter chrome-extension zip",
+    "extension:watch": "pnpm --filter chrome-extension watch",
+    "extension:e2e": "pnpm --filter chrome-extension-e2e e2e",
+    "extension:release": "pnpm --filter chrome-extension release",
+    "extension:publish": "pnpm --filter chrome-extension publish-to-store",
+    "website:build": "pnpm --filter website build",
+    "website:start": "pnpm --filter website start",
+    "website:release": "pnpm --filter website release",
+    "build": "pnpm website:build && pnpm extension:build",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
-    "lint:fix": "yarn lint --fix",
-    "release": "yarn website:release && yarn extension:release"
+    "lint:fix": "pnpm lint --fix",
+    "test": "echo \"No unit tests \"",
+    "test:dts": "echo \"No dts tests \"",
+    "release": "pnpm website:release && pnpm extension:release"
   },
-  "workspaces": [
-    "packages/**"
-  ],
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@commitlint/cli": "^17.6.3",
-    "@commitlint/config-conventional": "^17.6.3",
-    "@semantic-release/changelog": "^6.0.1",
-    "@semantic-release/git": "^10.0.1",
```

Comment:
You removed semantic release plugins from deps, but release process still the same as I understand.